examples/ci-tests/i2c-master-tx/README.md

```diff
@@ -1,22 +1,63 @@
-# Hardware CI Tests
+# I2C Master TX Test
 
-## Run Python Test
+## Test Description
 
-`test.py` provides basic test that sends message from master to slave, and waits for a 12 second period of master sending messages, but captures the first message sent. The message sent from master will be printed over a 12 second period. If message sent from master is the same as the expected message to be sent, then test will pass.
+`test.py` provides the following tests:
+
+### I2CMasterTxTest
+
+This sends message from the device-under-test as I2C Master to the RPi as I2C Slave.
+
+  1. The device-under-test configures itself as I2C master with address 0x40 and awaits button press.
+  1. The test harness presses a button (GPIO toggle) on device-under-test to indicate the device-under-test to send a message under master.
+  1. The test harness issues an I2C read transaction of 16 bytes and awaits master to send data.
```

Comment:
```suggestion
  1. The test harness issues an I2C read of 16 bytes and awaits the device-under-test as master to send data over the I2C bus until the local receive buffer is full.
```

It's a little confusing sometimes what's happening on the bus versus what is happening on the device. I think people usually think of "transactions" as bus transactions. I tried to update this language to be a little clearer, especially around who is actually causing bits to be sent on the bus when.